packages/paste-core/components/badge/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@twilio-paste/badge",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "alpha",
+  "description": "TODO update.",
```

Comment:
Just wanna make sure this doesn't get missed! Here's what we have down in the newsletter:
>A small chunk of information that describes an object in a visually highlighted way.

Reply:
This does go out in the npm registry, so we should probably add something here for now.